docs/docker-compose.yml

```
@@ -1,12 +1,59 @@
 services:
   mkdocs:
     image: telestion/mkdocs
+    depends_on:
+      backend-features:
+        condition: service_completed_successfully
+      frontend-react:
+        condition: service_completed_successfully
     ports:
       - 4044:4044
```

Please specify host binds explicitly. For example:

```suggestion
      - "0.0.0.0:4044:4044"
```